packages/alfa-rules/src/sia-r70/rule.ts

```diff
@@ -0,0 +1,140 @@
+import { Rule, Diagnostic } from "@siteimprove/alfa-act";
+import { Document, Element, Namespace } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Page } from "@siteimprove/alfa-web";
+import { List } from "@siteimprove/alfa-list";
+import { Iterable } from "@siteimprove/alfa-iterable";
+
+import { expectation } from "../common/expectation";
+import { hasChild, isRendered, isDocumentElement } from "../common/predicate";
+
+const { isElement, hasName, hasNamespace } = Element;
+const { and, test } = Predicate;
+
+const deprecated = [
+  "applet",
+  "basefont",
+  "bgsound",
+  "big",
+  "blink",
+  "center",
+  "content",
+  "dir",
+  "font",
+  "frame",
+  "frameset",
+  "hgroup",
+  "image",
+  "keygen",
+  "marquee",
+  "menuitem",
+  "nobr",
+  "noembed",
+  "noframes",
+  "plaintext",
+  "rb",
+  "rtc",
+  "shadow",
+  "spacer",
+  "strike",
+  "tt",
+  "xmp",
+];
+
```

Comment:
Pretty composition is pretty powerful:

```suggestion
/**
 * {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element#obsolete_and_deprecated_elements}
 */
const isDeprecated = hasName(
  "acronym",
  "applet",
  "basefont",
  "bgsound",
  "big",
  "blink",
  "center",
  "content",
  "dir",
  "font",
  "frame",
  "frameset",
  "hgroup",
  "image",
  "keygen",
  "marquee",
  "menuitem",
  "nobr",
  "noembed",
  "noframes",
  "plaintext",
  "rb",
  "rtc",
  "shadow",
  "spacer",
  "strike",
  "tt",
  "xmp",
);

```